src/frontend/Felles/Knapper/FjernKnapp.tsx

```diff
@@ -3,16 +3,18 @@ import React from 'react';
 import { Button } from '@navikt/ds-react';
 
 const FjernKnapp: React.FC<{
-    onClick: () => void;
-    knappetekst?: string;
+    className?: string;
     ikontekst?: string;
-}> = ({ onClick, knappetekst, ikontekst }) => {
+    knappetekst?: string;
+    onClick?: () => void;
```

Comment:
Er det riktig at disse skal være optional? 👀

Reply:
Ja, dette er riktig. Dersom knappen er av typen `submit` eller `reset` behøver den ingen onClick for å fyre av en funksjon